lib/facter/puppetdb_version.rb

```diff
@@ -0,0 +1,13 @@
+Facter.add(:puppetdb_version) do
+  confine { Facter::Util::Resolution.which('puppetdb') }
+
+  setcode do
+    output = Facter::Core::Execution.execute('puppetdb --version')
+
+    if output.nil?
+      nil
```

Comment:
I think we don't need to return nil since we confine on the puppetdb binary?

Reply:
thought i add this in case of `puppetdb` returns some error.

Reply:
removed it.